python/lsst/cp/pipe/ptc.py

```diff
@@ -623,55 +735,76 @@ def calculateLinearityResidualAndLinearizers(self, exposureTimeVector, meanSigna
 
         Returns
         -------
-        c0: `np.float`
+        polynomialLinearizerCoefficients : `list` of `np.float`
+            Coefficients for LinearizePolynomial, where corrImage = uncorrImage + sum_i c_i uncorrImage^(2 +
+            i).
+            c_(j-2) = -k_j/(k_1^j) with units (DN^(1-j)). The units of k_j are DN/t^j, and they are fit from
+            meanSignalVector = k0 + k1*exposureTimeVector + k2*exposureTimeVector^2 +...
+                             + kn*exposureTimeVector^n, with n = "polynomialFitDegreeNonLinearity".
+            k_0 and k_1 and degenerate with bias level and gain, and are not used by the non-linearity
+            correction. Therefore, j = 2...n in the above expression (see `LinearizePolynomial` class in
+            `linearize.py`.)
+
+        c0 : `np.float`
             Coefficient for LinearizeSquared, where corrImage = uncorrImage + c0*uncorrImage^2.
-            c0 ~ -k2/(k1^2), where k1 and k2 are fit from
+            c0 = -k2/(k1^2), where k1 and k2 are fit from
             meanSignalVector = k0 + k1*exposureTimeVector + k2*exposureTimeVector^2 +...
                                + kn*exposureTimeVector^n, with n = "polynomialFitDegreeNonLinearity".
 
-        linearizerTableRow: list of `np.float`
+        linearizerTableRow : list of `np.float`
```

Comment:
Use back ticks. I think `np.float` and `float` are interchangeable so do they really need to be numpy floats?